src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1375,6 +1376,7 @@ void OrbitMainWindow::ShowSourceCode(const std::filesystem::path& file_path, siz
   orbit_code_viewer::Dialog code_viewer_dialog{this};
 
   code_viewer_dialog.SetEnableLineNumbers(true);
+  code_viewer_dialog.SetEnableSampleCounters(true);
```

Comment:
I think this should only be enabled when `maybe_code_report.has_value()` is true. Can you move it down to line 1390?

Reply:
nice catch!